test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1150 @@
+| expirationTime          | int64                                            | optional          | Epoch time in seconds                                                                                                                       |
+| autoRenewAccount        | string                                           | optional          |                                                                                                                                             |
+| autoRenewPeriod         | int64                                            | optional          | Units of seconds                                                                                                                            |
+| memo                    | string                                           | optional          |                                                                                                                                             |
+| tokenType               | string                                           | optional          | MUST be one of `ft` (fungible token) or `nft` (non-fungible token)                                                                          |
+| supplyType              | string                                           | optional          | MUST be one of `infinite` or `finite`                                                                                                       |
+| maxSupply               | int64                                            | optional          |                                                                                                                                             |
+| feeScheduleKey          | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
+| customFees              | list<[json object](../customFee.md)>             | optional          |                                                                                                                                             |
+| pauseKey                | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
+| metadata                | string                                           | optional          | Hex-encoded bytes of the metadata                                                                                                           |
+| metadataKey             | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
```

Comment:
This is just a public key, or a serialized key message.  This should never be a private key.